### PR TITLE
removing RWX un-used references (s3). Updating memcached images references

### PR DESCRIPTION
**Pull request by maneta**
This solves the issues stated on https://issues.jboss.org/browse/THREESCALE-994 and updates the Memcached namespace on amp

### CODE REVIEW

**Comment by maneta** [comment]
closes #22 and #19

File: amp/amp-s3.yml [hunk 1]
```diff
@@ -828,7 +817,7 @@ objects:
         containers:
         - args:
           env:
-          image: 3scale-amp20/memcached:1.4.15-8
+          image: registry.access.redhat.com/3scale-amp20/memcached:1.4.15-8
```
Comment (hallelujah) [inline amp/amp-s3.yml]: We should use `1.4.15` tag. Not including the release version

**Comment by hallelujah** [comment]
And the productized template needs to be respun too

**Comment by maneta** [comment]
@hallelujah done in https://github.com/3scale/openshift-templates/pull/304

**Comment by alanmoran** [comment]
Can we have the 2.2GA release updated to use this recent commit as our documentation does have references to this github repo?

**Comment by maneta** [comment]
yes sure @alanmoran !

**Comment by maneta** [comment]
Done @alanmoran

**Comment by alanmoran** [comment]
Thanks @maneta

**Pull request merged**